src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_NullableHelpers.vb

```diff
@@ -205,6 +201,46 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return New BoundBadExpression(expr.Syntax, LookupResultKind.NotReferencable, ImmutableArray(Of Symbol).Empty, ImmutableArray.Create(expr), expr.Type.GetNullableUnderlyingType(), hasErrors:=True)
         End Function
 
+        Private Function NullableValueOrDefaultOpt(expr As BoundExpression) As BoundExpression
```

Comment:
>NullableValueOrDefaultOpt [](http://example.com/codeflow?start=25&length=25)

I suggest adjusting behavior of the function above based on a new parameter. Otherwise, we are missing handling for scenarios that simply need unwrapping constructor or conversion

Reply:
There was already an overload that took 1 boolean parameter, so I had to rename it first